### ISSUE TITLE
DB migrator

### ISSUE THREAD

**Issue by avivsarig** [avivsarig/beerpy#19]
Migrator: a tool to bring a database from state A to state B. we will write a simple migrator (there are state of the art migrators with lots of features).
migration files are text files with SQL commands inside that will perform operations on the DB (create tables, columns, change data types, add data, etc.).
we will denote the files as 000X_<some_indicative_name>.sql like 0003_beers_tables.sql
all migration files will have their own directory migrations in a folder in the project.
we will manage the state of the db in a database table called migrations_state that will have 3 columns (id, timestamp, migration_id)
the migrator will read the state from the db table, and will run all files from this state until covering all migration files.
after each successful migration we will update the migrations_state table
so basically the migrator should have the API:
migrate()

**Comment by avivsarig** [comment]
#23
